packages/input-option/src/InputOption/InputOption.story.tsx

```diff
@@ -91,19 +94,47 @@ export default meta;
 export const LiveExample: StoryFn<
   InputOptionProps & InputOptionContentProps
 > = (props: InputOptionProps & InputOptionContentProps) => {
-  const { leftGlyph, rightGlyph, description, ...rest } = props;
+  const { leftGlyph, rightGlyph, description, renderedContext, ...rest } =
+    props;
   return (
-    <InputOption {...rest}>
-      <InputOptionContent
-        leftGlyph={leftGlyph ? <Icon glyph={leftGlyph as string} /> : undefined}
-        rightGlyph={
-          rightGlyph ? <Icon glyph={rightGlyph as string} /> : undefined
-        }
-        description={description}
-      >
-        Some text
-      </InputOptionContent>
-    </InputOption>
+    <div style={{ display: 'flex', gap: '16px' }}>
+      <div>
+        <pre>Form</pre>
```

Comment:
nit: Should probably be separate stories, but this is fine for demo

Reply:
We can have separate stories, but if we ever want it on .design, we'd probably want to show both in one story for the LiveExample. Similar case to EmptyState